### DISCUSSION TITLE
Zigbee within multiple locations with 1 m2z server

### DISCUSSION THREAD

**Discussion by Leodien** [Koenkk/zigbee2mqtt#10702] [General]
I run Z2M in my house on a Raspberry with USB stick. my garage is too far from the house to use zigbee components in combination with the M2Z server. There is a LAN connection. What options do I have to use zigbee in the garage?
thanks in advance

**Comment by TommySharpNZ** [comment]
I have this exact same situation!
Maybe someone could explain how something like this might help?
https://smartlight.me/smart-home-devices/zigbee-devices/smlight-zigbee-lan-adapter-slzb-05en

**Reply by m0wlheld** [reply]
Guess you need one Z2M instance per coordinator device and each coordinator spawns it‘s own Zigbee network. 

Both can use the same MQTT server, if you choose different base topics.
